### PR TITLE
4.0.0/stability

### PR DESCRIPTION
**Pull request by rart**
- Add GraphQL icon
- Item actions clean up
- Fix found runtime exception due to unhandled optional prop
- Fix found runtime exception due to path sometimes not containing "/index.xml"
- Avoid runtime exceptions in item inspection functions supplied with null item
- Fix Graphql icon copyright
- Falling back to default framework input styles
- Adjusting customised input styles
- Add missing translations

### CODE REVIEW

File: ui/app/src/components/Icons/GraphQL.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2007-2020 Crafter Software Corporation. All Rights Reserved.
```
Comment (sumerjabri) [inline ui/app/src/components/Icons/GraphQL.tsx]: Wrong copyright notice date. Please copy/paste latest from Slack.

File: ui/app/src/components/Icons/GraphQL.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
...6 18.8425 18.8343C17.9043 18.2949 17.5841 17.0927 18.1234 16.1545C18.6627 15.2163 19.8649 14.8961 20.8031 15.4354C21.747 15.9803 22.0672 17.177 21.5222 18.1152ZM5.87062 9.07588C5.3313 10.0141 4.12906 10.3343 3.19086 9.79497C2.25266 9.25565 1.93244 8.05341 2.47176 7.11521C3.01108 6.17701 4.21333 5.85679 5.15152 6.39611C6.08972 6.94105 6.40995 8.13768 5.87062 9.07588ZM2.47738 18.1152C1.93806 17.177 2.25828 15.9803 3.19648 15.4354C4.13468 14.8961 5.3313 15.2163 5.87624 16.1545C6.41556 17.0927 6.09534 18.2893 5.15714 18.8343C4.21333 19.3736 3.0167 19.0534 2.47738 18.1152ZM18.129 9.07588C17.5897 8.13768 17.9099 6.94105 18.8481 6.39611C19.7863 5.85679 20.9829 6.17701 21.5279 7.11521C22.0672 8.05341 21.747 9.25003 20.8088 9.79497C19.8706 10.3343 18.6683 10.0141 18.129 9.07588ZM11.9998 23.6151C10.9155 23.6151 10.0391 22.7387 10.0391 21.6545C10.0391 20.5702 10.9155 19.6938 11.9998 19.6938C13.0841 19.6938 13.9605 20.5702 13.9605 21.6545C13.9605 22.7331 13.0841 23.6151 11.9998 23.6151ZM11.9998 5.53657C10.9155 5.53657 10.0391 4.66017 10.0391 3.5759C10.0391 2.49164 10.9155 1.61523 11.9998 1.61523C13.0841 1.61523 13.9605 2.49164 13.9605 3.5759C13.9605 4.66017 13.0841 5.53657 11.9998 5.53657Z" />
```
Comment (sumerjabri) [inline ui/app/src/components/Icons/GraphQL.tsx:28]: Can this line be broken up?

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: ui/app/src/components/Icons/GraphQL.tsx (hunk 1 above)
Comment (rart) [reply ui/app/src/components/Icons/GraphQL.tsx]: Done.

File: ui/app/src/components/Icons/GraphQL.tsx (hunk 2 above)
Comment (rart) [reply ui/app/src/components/Icons/GraphQL.tsx:28]: I'm not sure. Not worth the effort anyway as these are sources generated by the design tool.

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**